eo-maven-plugin/src/main/java/org/eolang/maven/Moja.java

```diff
@@ -147,30 +146,35 @@ private static Collection<Field> fields(final Class<?> cls) {
     }
 
     /**
-     * Take a field.
-     * @param mojo The class
-     * @param name Field name
-     * @return Field
+     * Init a field.
+     * @param clazz The mojo class
+     * @param mojo The mojo
+     * @param entry Field name and value
+     * @throws java.lang.IllegalAccessException If can't set field.
      */
-    private Field field(final Class<?> mojo, final String name) {
-        Field field;
+    private void initField(
+        final Class<?> clazz,
+        final AbstractMojo mojo,
+        final Map.Entry<String, Object> entry
+    ) throws IllegalAccessException {
+        final String name = entry.getKey();
         try {
-            field = mojo.getDeclaredField(name);
+            final Field field = clazz.getDeclaredField(name);
+            field.setAccessible(true);
+            field.set(mojo, entry.getValue());
         } catch (final NoSuchFieldException ex) {
-            final Class<?> parent = mojo.getSuperclass();
+            final Class<?> parent = clazz.getSuperclass();
             if (parent == null) {
```

Comment:
@volodya-lombrozo shouldn't we throw an exception in this case as it was before? Otherwise we just hide the problem.

Reply:
@mximp It helps me to configure `FakeMaven`. If some of the properties are absent in `Mojo`, then the `NoSuchFieldException ` will thrown. Of course, I can write the logic inside `FakeMaven` in order to check whether the property is allowed
for particular `Mojo`, but it is quite a verbose solution and it doesn't worth it. Moreover, since all tests are pass, we can conclude that we don't break anything by that changes.

Reply:
@volodya-lombrozo ok.. I believe in this case there is no point in warning here.. What is the expected reaction to it?

Reply:
@mximp expected reaction to what? As you can see - we just ignore field if it isn't found in the class.

Reply:
> @mximp expected reaction to what? As you can see - we just ignore field if it isn't found in the class.

@volodya-lombrozo why do we print warning then?

Reply:
@mximp  in order to inform a programmer that it is unusual situation

Reply:
@volodya-lombrozo we will always have such warnings as the list of properties is shared for all mojos and contains a superset of props. With such a case it's normal that some of the props will not be applicable for some mojos. There should be no warning in this case.

Reply:
@mximp updated. Please, check